release-notes.txt

```diff
@@ -1,3 +1,619 @@
+RIOT-2023.04 - Release Notes
+============================
+RIOT is a multi-threading operating system which enables soft real-time
+capabilities and comes with support for a range of devices that are typically
+found in the Internet of Things: 8-bit and 16-bit microcontrollers as well as
+light-weight 32-bit processors.
+
+RIOT is based on the following design principles: energy-efficiency, soft
+real-time capabilities, small memory footprint, modularity, and uniform API
+access, independent of the underlying hardware (with partial POSIX compliance).
+
+RIOT is developed by an international open-source community which is
+independent of specific vendors (e.g. similarly to the Linux community) and is
+licensed with a non-viral copyleft license (LGPLv2.1), which allows indirect
+business models around the free open-source software platform provided by RIOT.
+
+
+About this release
+==================
+
+The 2023.04 release includes:
+
+- Mass Storage Class Support in USBUS
+- Initial support for libSCHC
+- FlashDB support
+- BLE downlink in GNRC Border Router
+- New boards:
+  - esp32c3-wemos-mini
+  - esp32s2-lilygo-ttgo-t8
+  - esp32s2-wemos-mini
+  - esp32s3-pros3
+  - rpi-pico-w
+  - sipeed-longan-nano
+- New device drivers:
+  - Black/white e-Paper/e-Ink display
+  - RFID reader (MFRC522)
+  - ST VL53L1X Time-of-Flight sensor
+
```

Comment:
- strings (and other const data) no longer needs to be copied to RAM on AVR with `flash_utils`

Reply:
Is congestion control for 6LoWPAN SFR noteworthy?

Reply:
done!